marklogic-unit-test-modules/src/main/ml-modules/root/test/test-helper.xqy

```diff
@@ -173,7 +173,7 @@ declare function test:get-modules-file($file as xs:string, $format as xs:string?
         xdmp:unquote($doc)
 };
 
-declare variable $local-url as xs:string := xdmp:get-request-protocol() || "://localhost:" || xdmp:get-request-port();
+declare variable $local-url as xs:string := xdmp:get-request-protocol() || "://%%mlHost%%:" || xdmp:get-request-port();
```

Comment:
Hey @dmcassel, is the thought that this will be replaced by ml-gradle during the deployment step?

I'm not sure that will always age correctly, as sometimes the DNS entry for installation may not be the same DNS entry used by the cluster to reference itself.

Reply:
Yes, that's the thought. `mlHost` is the configured host name, not necessarily the hostname used by the cluster to refer to itself, which is what I think you'd get from `xdmp.hostName`.

Reply:
I assume you've tested this out to make sure on a deployed environment ml-gradle does the proper token replacement with an `mlDeploy`?

If so, I'll go ahead and accept the change and merge in the PR.